apps/condo/domains/common/components/BaseSearchInput/index.tsx

```diff
@@ -141,6 +141,7 @@ export const BaseSearchInput = <S extends string>(props: ISearchInput<S>) => {
             filterOption={false}
             autoClearSearchValue={false}
             defaultActiveFirstOption={false}
+            {...restSelectProps}
```

Comment:
?

Reply:
I need to forward a `style` prop into `Select` child component of `BaseSearchInput`.